modules/configure-pagerduty.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-pagerduty.adoc
+:_module-type: PROCEDURE
+[id="configure-pagerduty_{context}"]
+= Configuring PagerDuty
+
+[role="_abstract"]
+Start by creating a new service in PagerDuty, and get the integration key.
```

Comment:
Suggestion for intro: "Start integrating with PagerDuty by creating a new service and by getting the integration key."